tests/network/expose.go

```diff
@@ -817,3 +807,13 @@ func createAndWaitForJobToSucceed(jobFactory func(host, port string) *batchv1.Jo
 	By("Waiting for the job to report a successful connection attempt")
 	return job.WaitForJobToSucceed(jobInstance, time.Duration(120)*time.Second)
 }
+
+func runJobsAgainstService(svc *k8sv1.Service, namespace string, jobFactories ...func(host, port string) *batchv1.Job) {
+	serviceIPs := svc.Spec.ClusterIPs
+	for _, jobFactory := range jobFactories {
+		for ipOrderNum, ip := range serviceIPs {
+			servicePort := fmt.Sprint(svc.Spec.Ports[0].Port)
```

Comment:
nit:

I prefer `strconv.Itoa` over `fmt.Sprint` for stringfy numbers. But here, port is int32, so we'll have to cast it:
```go
servicePort := strconv.Itoa(int(svc.Spec.Ports[0].Port))
```

so maybe it not so nice. your call.

Reply:
It's not much better so I prefer to keep it like this

Reply:
I agree with @nunnatsa , `Itoa` is focus on converting integers to strings and is the prefer function to work with in such cases. Using the `fmt` is IMO a misuse.

I could argue it is also much slower to do the conversion with the formatter, but that has little meaning in these tests.

Reply:
If you both think it's better then it's enough for me. 
DONE